python/pyspark/sql/column.py

```diff
@@ -298,19 +299,34 @@ def isin(self, *cols):
     isNotNull = _unary_op("isNotNull", "True if the current expression is not null.")
 
     @since(1.3)
-    def alias(self, *alias):
+    def alias(self, *alias, **kwargs):
         """
         Returns this column aliased with a new name or names (in the case of expressions that
         return more than one column, such as explode).
 
+        Optional ``metadata`` keyword argument can be passed when aliasing a single column.
```

Comment:
Excellent, so I think maybe using the `:param:` docstring notation here would be make sense (as in the other functions).